src/main.c

```diff
@@ -35,7 +33,17 @@ bool promptFileName(char* fileName) {
 	size_t bufsize = 100;
 
 	/*scan for filename*/
-	getline(&fileName,&bufsize, stdin);
+	
```

Comment:
@thomasterhune I know you had found `getline` as a solution for enter/no input but, unfortunately, I don't think we can compile in `ansi` because `getline` was added more recently. I believe we will have to find another solution to that problem.

Reply:
Okay i will figure something out